tests/Makefile.tests_common

```diff
@@ -8,7 +8,7 @@ ifneq (,$(wildcard $(CURDIR)/tests*/.))
 
   # do the same for Kconfig builds
   ifeq (1,$(TEST_KCONFIG))
-    KCONFIG_ADD_CONFIG += $(RIOTBASE)/tests/test_utils.config
+      KCONFIG_ADD_CONFIG += $(RIOTBASE)/tests/test_utils.config
```

Comment:
unrelated

Reply:
Ah, leftover... Fixed and directly ameded.